### PR TITLE
Build fixes

### PR DESCRIPTION
**Pull request by jdforrester**

### CODE REVIEW

File: test/jquery.i18n.test.js [hunk 1]
```diff
@@ -392,15 +392,15 @@
...
+	    var done = assert.async();
 		$.when(
 			i18n.load( 'i18n/fallback', 'uk' )
 		).then( function () {
-			QUnit.start();
 			i18n.locale = 'uk';
 			assert.strictEqual( i18n.locale, 'uk', 'Locale is uk' );
 			assert.strictEqual( $.i18n( 'message_1' ), 'ONE',
 				'Message loaded from fallback locale English' );
+	        done();
```
Comment (Krinkle) [inline test/jquery.i18n.test.js]: Legacy start/stop toggled a semaphore that was checked at the end of the test callback execution. `assert.async()` instead is meant to mark the end of the execution. Moving it lower is indeed required. However given the possibility of a failure scenario here, it should probably be passed to `.always()` not `.then()`.

In this case though, I'd recommend simply removing it alltogether and returning the promise from `$.when().then()` from the callback directly to to QUnit. Which will naturally wait for both `done` or `fail`, and also mark the test as failed if the promise is rejected.

**Comment by santhoshtr** [comment]
I am bit confused by this PR. What exactly is the proposed change?

**Comment by jdforrester** [comment]
First change was 47a623a, last change is 0dbd4c3, and GitHub is sadly fundamentally not suited to rebases.

**Comment by Krinkle** [comment]
I've rebased it on top of latest master. The tests failed because the setup/teardown stages were no longer happening. These hooks have been renamed to beforeEach/afterEach in QUnit 2.x. I've switched those as part of the "Bump qunitjs from 1.23.1 to 2.4.1" commit. The tests now pass.

**Pull request merged**